Demo/src/main/java/com/braintreepayments/demo/CreateTransactionFragment.java

```diff
@@ -97,6 +100,9 @@ public void failure(RetrofitError error) {
             transactionRequest = new TransactionRequest(amount, nonceString, unionPayMerchantAccountId);
         } else {
             String merchantAccountId = Settings.getMerchantAccountId(activity);
+            if (merchantAccountId == null && nonce instanceof PayPalNativeCheckoutAccountNonce) {
+                merchantAccountId = PAYPAL_NATIVE_CHECKOUT_CLIENT_ID;
```

Comment:
since we create the ba token with this client id we need to execute the transaction with it as well